src/Runner.Worker/StepsRunner.cs

```diff
@@ -136,7 +135,7 @@ public async Task RunAsync(IExecutionContext jobContext)
                             // Test the condition again. The job was canceled after the condition was originally evaluated.
                             jobCancelRegister = jobContext.CancellationToken.Register(() =>
                             {
-                                // mark job as cancelled
+                                // Mark job as cancelled
```

Comment:
Spent a while skimming this file.... formatted all the comments consistently so it skims easier